AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -94,6 +94,8 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+jobs.golem.de#$#body { overflow: visible !important; }
+jobs.golem.de#$#div[class^="jwtpl-jmpt-cookieNotice"] { display: none !important; }
```

Comment:
Without accepting cookies, you won't see the information here https://jobs.golem.de/Job/Software-Entwickler-Java-C-m-w-d.561704235.html
I'll fix it.